packages/web-components/fast-components/src/disclosure/index.ts

```diff
@@ -0,0 +1,97 @@
+import { attr, customElement } from "@microsoft/fast-element";
+import { Disclosure, DisclosureTemplate as template } from "@microsoft/fast-foundation";
+import { DisclosureStyles as styles } from "./disclosure.styles";
+/**
+ * Types of anchor appearance.
+ * @public
+ */
+export type DisclosureAppearance = "accent" | "hypertext";
```

Comment:
I'm not sure we'll want this to render inline with text - we should likely update all references of `hypertext` to lightweight. The intent of hypertext is to style and have it rendered inline with text. Lightweight is more intended as a block-level element with a text-only appearance.

Reply:
Yes, I think that would be more sensible to change it to `lightweight`. Will do 😄